bulbs/content/managers.py

```diff
@@ -0,0 +1,105 @@
+"""Generic manager for all indexable content and common queries."""
+from django.conf import settings
+from django.utils import timezone
+
+from djes.models import IndexableManager
+from elasticsearch_dsl import filter as es_filter
+from polymorphic import PolymorphicManager
+
+from .filters import (
+    AllSponsored, Authors, Evergreen, FeatureTypes, Published, Status, Tags, VideohubChannel
+)
+
+
+class ContentManager(PolymorphicManager, IndexableManager):
+    """a specialized version of `djes.models.SearchManager` for `bulbs.content.Content`."""
+
+    def evergreen(self, **kwargs):
+        """
+        Search containing any evergreen piece of Content.
+
+        :included_channel_ids list: Contains ids for channel ids relevant to the query.
+        :excluded_channel_ids list: Contains ids for channel ids excluded from the query.
+        """
+        eqs = self.search(**kwargs)
+        eqs = eqs.filter(Evergreen())
+        included_channel_ids = kwargs.get("included_channel_ids", [])
+        excluded_channel_ids = kwargs.get("exncluded_channel_ids", [])
```

Comment:
Why not pass these as explicit keyword arguments to this def, instead of hiding them in **kwargs? Also it would make it explicit that these are custom args and not to be passed to self.search() above.